theme6/source/stylesheets/site.css.scss

```diff
@@ -0,0 +1,169 @@
+
+
+
+/*==========Stylesheet DESKTOP========*/
+    footer {
+      bottom: 0;
+      display: inline;
+      width: 100%;
+
+    }
+
+    .information{justify-content: center;}
+
+    li{display: inline;}
+
+    .socialmedia{
+        color: #72808C;
+        font-family:'Montserrat', sans-serif;
+        font-size: 18px;
+        line-height: 21px;
+    }
+
+    .sm {
+        height: 35px;
+        width: 93px;
+        border: 2px solid #72808C;
+        border-radius: 3px;
+    }
+    #fb{
+      background: url("../images/fb.svg");
+      background-position: center center;
+      background-repeat: no-repeat;
+      margin-right: 23px;
+    }
+    #twt{
+        background: url("../images/twt.svg");
+        background-position: center center;
+        background-repeat: no-repeat;}
+    #g{
+        background: url("../images/g+.svg");
+        background-position: center center;
+        background-repeat: no-repeat;
+        margin-left: 23px;
+      }
+
+    .newsletter {
+      color: #72808C;
+      font-family: 'Montserrat', sans-serif;
+      font-size: 18px;
+      line-height: 21px;
+      text-align: center;
+    }
+
+    footer .box-social {
+      align-content: center;
+      width: 350px;
+    }
+
+    footer .box-social button:hover {
+      cursor: pointer;
+    }
+
+    footer .contenedorsocial {
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+      width: 66%;
+    }
+
+    footer .contenedorsocial .logo {
+      margin-left: 96px;
+      margin-right: 96px;
+    }
+
+    footer .box-social p{
+      text-align: center;
+    }
+    footer .contenedornews{
+      width: 33%;
+      display: inline-flex;
+    }
+    footer .newsletter input {
+      height: 35px;
+      width:300px;
+      justify-content: center;
+    }
+
+    footer .information ul li a{
+      height: 22px;
```

Comment:
Properties should be ordered align-items, color, font-family, font-size, height, line-height, margin-right, text-decoration, width